evmcore/state_processor.go

```diff
@@ -186,6 +190,42 @@ func runTransactions(
 	return processed
 }
 
+const (
+	// RlpEncodedMaxHeaderSizeInBytes is an upper bound of the EVM block header size
+	// used for block size calculations.
+	RlpEncodedMaxHeaderSizeInBytes = uint64(1024)
+
+	// RlpEncodedInternalTransactionSizeInBytes is an upper bound for the size of
+	// RLP-encoded internal transactions included in the block for block size
+	// calculations.
+	// Transactions during the setup of the chain are not considered,
+	// it is unlikely to have a lot of payload transactions during the setup phase.
+	// Cheating validator transactions have a size of around 112 bytes.
+	// Epoch Seal internal transaction have a size of around 10808 bytes.
+	// Epoch Seal Validator internal transaction have a size of around 2744 bytes.
+	RlpEncodedInternalTransactionSizeInBytes = uint64(16384)
```

Comment:
From the documentation it is not clear where this number is coming from. It is not the sum of the provided values. I presume it is intended to be all cases, rounded up to 16 KiB, so maybe state this in the comment and define it like
```suggestion
	RlpEncodedInternalTransactionSizeInBytes = uint64(16 * 1024)  // 16 KiB
```